model.js

```diff
@@ -11,4 +11,15 @@ exports.fetchTopics = () => {
 
 exports.fetchApiInformation = () => {
     return apiInfo
+}
+
+exports.fetchArticleWithCorrectId = (articleId) => {
+  const query = format(`SELECT * FROM articles WHERE article_id = ${articleId};`);
+  return db.query(query).then((data)=> {
```

Comment:
There is no need to use format, you are open to SQL injection, what can you do to prevent SQL injection?